LocalWiki.php

```diff
@@ -29,10 +29,10 @@ function onBeforePageDisplay( OutputPage $out ) {
 
 		break;
 	case 'gratisdatawiki':
-		$wgHooks['overridePageMetaTags'][] = 'onoverridePageMetaTags';
+		$wgHooks['OverridePageMetaTags'][] = 'onOverridePageMetaTags';
```

Comment:
No, that's not the first letter being lowercase

Reply:
> No, that's not the first letter being lowercase

Oh, like the first one `overridePageMetaTags` then the remaining `OverridePageMetaTags`?

Reply:
> Yes

Okay, Done!